tq42/project.py

```diff
@@ -109,7 +109,11 @@ def set_friendly_name(self, friendly_name: str) -> Project:
         For details, see
         https://terra-quantum-tq42sdk-docs.readthedocs-hosted.com/en/latest/Python_Developer_Guide/Setting_Up_Your_Environment.html#setting-friendly-names-for-projects-and-experiments
         """
-        return self.update(name=friendly_name)
+        self.update(name=friendly_name)
+        print(
+            f"Set the friendly name '{ friendly_name   }' for the project id '{ self.id   }'"
```

Comment:
formatting

Reply:
weird, the `format` automatic check did not complain about that.

I add a space and re-commit and it worked.